src/com/talar/server/Server.java

```diff
@@ -0,0 +1,258 @@
+package com.talar.server;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.InetAddress;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+public class Server {
+
+        private int port = 0;
+        private ServerSocket server = null;
+        private BufferedInputStream bis = null;
+        private BufferedOutputStream bos = null;
+
+        public Server() {
+
+        }
+
+        public void start() {
+                Socket socket = null;
+                boolean loop = false;
+                
+                try {
+                        server = new ServerSocket(port);
+                        loop = true;
+                        
+                        while(loop) {
+                                System.out.println("Server started on port " + getPort());
+
+                                socket = server.accept();
```

Comment:
You might want to look at creating a new file like `ServerProcess` or something where you send it the `socket` as a parameter and move all the message processing code for that socket over there. Keeping `Server` just a place that accepts connections and creates new `ServerProcess` files to handle multiple clients.  The Server Process would need to be a `Thread`  it would extend thread class.